src/lesson-1-9-demo/ProductCatalog.jsx

```diff
@@ -6,13 +6,14 @@ const currencyFormatter = new Intl.NumberFormat("en-US", {
   currency: "USD",
 });
 
-const listProducts = (pageSize, cursor) => {
+const listProducts = async (pageSize, cursor) => {
```

Comment:
we convert this function to the `async/await` format by first adding the `async` keyword before the function